CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# OSIsoft Connector Code of Conduct
```

Comment:
It might be worth updating just 'OSIsoft Connector' to be 'Flexy OSIsoft Connector' or 'Ewon Flexy OSIsoft Connector' which gives a bit more indication that the connector is between Ewon Flexy and OSIsoft